lib/cli.js

```diff
@@ -25,6 +25,10 @@ Options:
 
 module.exports = function parseCLI(argv = process.argv.slice(2), help = HELP) {
 	argv = parseArgs(argv, {
+		boolean: ["watch", "fingerprint", "sourcemaps", "compact"],
+		default: {
+			config: "faucet.config.js"
```

Comment:
I just realized this doesn't belong here: The default value is already being set by `lib/config.js` - that's because we want that default to be the same whether you're using the CLI or the imperative API.
https://github.com/faucet-pipeline/faucet-pipeline-core/blob/d0a84d2e516840247e890eeaef040a8d8c7461f1/lib/config.js#L5

Reply:
I saw that. I just thought that with adding the default here as well, this mirrors the description in the help message above.

Reply:
I'm ok with reverting that part. Will you do that? Because I know you already have a new commit message drafted.

Reply:
Done: 8ffb8cacc42d19a033b1af1bff3b2644bc410933

Thank you!